### DISCUSSION TITLE
Awesome shadcn-ui

### DISCUSSION THREAD

**Discussion by corysimmons** [shadcn-ui/ui#2719] [Ideas]
I just found [this tag component](https://github.com/JaleelB/shadcn-tag-input) someone built to augment shadcn-ui and was wondering if there were a ton of these out there in the wild.

It might be cool to have a part of the README (or even just this Discussion pinned), to collect these neat little projects in.

---

I'll just keep a list here. Feel free to post stuff in the discussion and I'll update this list:

- https://ui.jln.dev - 10,000+ themes
- https://github.com/JaleelB/shadcn-tag-input - Input tag
- https://github.com/redpangilinan/credenza - Responsive modal with sheet on smaller sizes

**Comment by zhadavi** [comment]
+1 on the idea. Also thanks for letting me know about this. This is exactly what I was looking for!

**Comment by vaibhav-gupta5** [comment]
I just found this open source shadcn dashboard template. Might be helpful for the community:

- https://github.com/Tailwind-Admin/free-tailwind-admin-dashboard-template
